### ISSUE TITLE
[feature]: 

### ISSUE THREAD

**Issue by syl3n7** [makeplane/plane#1180]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

Is there any plan for integration with subversion? 

### Why should this be worked on?

Be able to use subversion with this can enable more organizations and schools to use your tool to replace good old redmine, for example :)

**Comment by Pdzly** [comment]
You mean this: https://subversion.apache.org/
With subversion or?

**Comment by rhea0110** [comment]
Hey @syl3n7, thank you for raising the feature request. Our team will brainstorm around it, and we will keep you updated on any developments.

Thanks!

**Comment by syl3n7** [comment]
> You mean this: https://subversion.apache.org/ With subversion or?

Yes, we use subversion at "my" university and some companies here use it too

**Comment by vihar** [comment]
Sorry for the slow reply here.

Subversion isn't on our roadmap — our integration work is Git-based, GitHub and GitLab today. The Redmine-replacement angle was a fair point to make, but SVN isn't somewhere we're heading.

Closing this out.
